udata/core/discussions/models.py

```diff
@@ -46,5 +46,5 @@ def person_involved(self, person):
     @property
     def external_url(self):
         return self.subject.url_for(
-            _anchor='discussion-{ id }'.format(id=self.id),
+            _anchor='discussion-{id}'.format(id=self.id),
```

Comment:
Yes, the spaces were used in the key resolution :( (we are too used too Jinja and Vue)